app/models/district.rb

```diff
@@ -12,7 +12,8 @@
 class District < ActiveRecord::Base
   belongs_to :state
   has_and_belongs_to_many :zip_codes
-  has_and_belongs_to_many :campaigns
+  has_one :representative, class_name: 'Legislator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.